.github/ISSUE_TEMPLATE/decision-record.md

```diff
@@ -10,13 +10,9 @@ Created on {YYYY-MM-DD} by {name and surname (@Github username)}.
 | Name | Description |
```

Comment:
Perhaps it would make sense to add as a comment even in this template the link to the decision making process (https://github.com/kyma-project/community/blob/master/governance.md)

Reply:
@natasab the link to the process is there in the comment

Reply:
@agolawska i meant would be good to put the link https://github.com/kyma-project/community/blob/master/governance.md in decision-record.md file (as comment)  :)

Reply:
@natasab ok, added :)